processor/transformprocessor/README.md

```diff
@@ -152,17 +161,55 @@ This is because contexts are nested: the efficiency comes because higher-level c
 ## Supported functions:
 
 Since the transform processor utilizes the OTTL's contexts for Traces, Metrics, and Logs, it is able to utilize functions that expect pdata in addition to any common functions. These common functions can be used for any signal.
-<!-- markdown-link-check-disable-next-line -->
+
 - [OTTL Functions](https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/pkg/ottl/ottlfuncs)
 
 In addition to OTTL functions, the processor defines its own functions to help with transformations specific to this processor:
 
+**Common functions**
+- [drop](#drop)
+
 **Metrics only functions**
 - [convert_sum_to_gauge](#convert_sum_to_gauge)
 - [convert_gauge_to_sum](#convert_gauge_to_sum)
 - [convert_summary_count_val_to_sum](#convert_summary_count_val_to_sum)
 - [convert_summary_sum_val_to_sum](#convert_summary_sum_val_to_sum)
 
+## drop
+
+`drop()`
+
+Drops the specified [Context's](#contexts) telemetry from the collector.  Best used with a condition, otherwise all the Context's telemetry would be dropped.
```

Comment:
```suggestion
Drops the specified [Context's](#contexts) telemetry from the pipeline.  Best used with a condition, otherwise all the Context's telemetry would be dropped.
```
Nitpick, but I think it's worth changing since if the telemetry has been fanned out across multiple pipelines, the transform processor will only mutate the telemetry within the pipeline(s) where it is defined.